app/controllers/user_applications_controller.rb

```diff
@@ -91,29 +90,44 @@ def destroy
     respond_with(@user_application)
   end
 
+  # View application
   def view
     @user_application = UserApplication.find(params[:id])
     authorize @user_application
     @user_information = @user_application.user.user_information
   end
 
+  # Accept application
   def accept
-    return change_status(params, "Accepted", "Application accepted successfully")
+    @user_application = UserApplication.find(params['id'])
+    authorize @user_application
+    volunteer = Volunteer.create(user_id: @user_application.user_id, volunteer_position_id: 0)
+    return change_status("Accepted", "Application accepted successfully")
```

Comment:
@SeanLF "return" keyword is implied if it's your last statement in the method. You can get rid of it here.